src/main/resources/db/migration/V7_alter_users_sequence.sql

```diff
@@ -0,0 +1 @@
+ALTER SEQUENCE users_seq INCREMENT BY 1;
```

Comment:
I don't see, where do you create the sequences?

Reply:
The problem with the sequences was that hibernate created sequences with increment by 50 and this was my attempt to solve this issue. I solved this issue with these annotations : 
e.g. @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "users_generator")
@SequenceGenerator(name="users_generator", sequenceName = "users_SEQ", allocationSize=1)